foundations/introduction/motivation_and_mindset.md

```diff
@@ -173,6 +173,7 @@ This section contains helpful links to related content. It isn’t required, so
 
 - [Managing inspiration and motivation](https://markmanson.net/do-something)
 - [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
+- [Becoming a TOP Success Story](https://dev.to/theodinproject/becoming-a-top-success-story-mindset-3dp2)
```

Comment:
This is a great idea to include this here. I say we move it to the top, for fun, but mainly because it is a specific article to help with TOP, for TOP, by TOP.
```suggestion
- [Becoming a TOP Success Story](https://dev.to/theodinproject/becoming-a-top-success-story-mindset-3dp2)
- [Managing inspiration and motivation](https://markmanson.net/do-something)
- [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
```

Reply:
I agree, i wanted to do it that way but thought it would be narcissistic of me 😅